### DISCUSSION TITLE
Show spaces from all displays

### DISCUSSION THREAD

**Discussion by itaysharir** [FelixKratz/SketchyBar#242] [Q&A] [answered]
How can i show all spaces from all displays instead on every display's spaces being in their relative display?
Thanks in advance!

**Comment by FelixKratz** [comment, marked as answer]
Hi, when creating the spaces you can set the option:
```bash
sketchybar --set <space_name> ignore_association=on
```
which essentially ignores the display association of the space components, showing all spaces on all displays.
